File: king-admin-java/src/main/java/com/oukingtim/mapper/SysRoleMenuMapper.java
package com.oukingtim.mapper;

import com.baomidou.mybatisplus.mapper.BaseMapper;
import com.oukingtim.domain.SysRoleMenu;

import java.util.List;

/**
 * Created by oukingtim
 */
public interface SysRoleMenuMapper extends BaseMapper<SysRoleMenu> {

    /**
     * 根据角色id查询菜单id
     * @param roleId
     * @return
     */
    List<String> selectMenuIdsByRoleId(String roleId);
}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/mapper/SysRoleMenuMapper.kt
package com.oukingtim.mapper

import com.baomidou.mybatisplus.mapper.BaseMapper
import com.oukingtim.domain.SysRoleMenu

/**
 * Created by oukingtim
 */
interface SysRoleMenuMapper : BaseMapper<SysRoleMenu> {
    /**
     * 根据角色id查询菜单id
     * @param roleId
     * *
     * @return
     */
    fun selectMenuIdsByRoleId(roleId: String): List<String>
}